variables.tf

```diff
@@ -103,6 +103,12 @@ variable "script_location" {
   description = "Specifies the S3 path to the script that is executed by this job"
 }
 
+variable "timeout" {
+  type        = number
+  default     = 2880
+  description = "The job timeout in minutes"
```

Comment:
Can we add a validation on this? Like >0 and < X?

Reply:
I would like to let the provider do the validation for these.